readthedocs/storage/azure_storage.py

```diff
@@ -30,6 +34,14 @@ def url(self, name, expire=None, http_method=None):  # noqa
         """
         return super().url(name, expire)
 
+    def exists(self, name, timeout=None):
```

Comment:
We don't actually use the `timeout` here, so probably shouldn't accept it.

Reply:
Meh! I was working in the other idea about passing a timeout here and forgot to remove it :/